docs/specs/build/toc/basics.yml

```diff
@@ -0,0 +1,596 @@
+---
+# 1. Normal file reference
+inputs:
+  docfx.yml: |
+    files:
+      include:
+        - docs/**/*.md
+        - docs/**/*.yml
+  index.md:
+  docs/toc.md: |
+    # [Index Reference](index.md)
+    ## Existing File Reference
+    ### [Reference Normal File 1](n1.md)
+    ### [Reference Normal File 2](../index.md)
+    ### [Reference Normal File 3](f1/n1.md?branch=master#row=4)
+    ### [Reference Normal File 4](~/docs/n1.md)
+  docs/index.md:
+  docs/n1.md:
+  docs/f1/n1.md:
+outputs:
+  index.json: |
+    { "content": "" }
+  docs/index.json: |
+    { "content": "" }
+  docs/n1.json: |
+    { "content": "" }
+  docs/f1/n1.json: |
+    { "content": "" }
+  docs/toc.json: |
+    [  
+      {  
+          "children":[  
+            {  
+                "children":[  
+                  {  
+                      "toc_title":"Reference Normal File 1",
+                      "href":"n1"
+                  },
+                  {  
+                      "toc_title":"Reference Normal File 2",
+                      "href":"../"
+                  },
+                  {  
+                      "toc_title":"Reference Normal File 3",
+                      "href":"f1/n1#row=4?branch=master"
+                  },
+                  {  
+                      "toc_title":"Reference Normal File 4",
+                      "href":"n1"
+                  }
+                ],
+                "toc_title":"Existing File Reference"
+            }
+          ],
+          "toc_title":"Index Reference",
+          "href":"."
+      }
+    ]
```

Comment:
toc.json is an object instead of an array

Reply:
ho god, thanks for the reminding!

---
In reply to: [185722737](https://github.com/dotnet/docfx/pull/2717#discussion_r185722737) [](ancestors = 185722737)